diff --git a/app/src/main/java/com/termux/app/TermuxActivity.java b/app/src/main/java/com/termux/app/TermuxActivity.java
index 280d937..48ce613 100644
--- a/app/src/main/java/com/termux/app/TermuxActivity.java
+++ b/app/src/main/java/com/termux/app/TermuxActivity.java
@@ -660,7 +660,7 @@ public final class TermuxActivity extends Activity implements ServiceConnection
                 workingDirectory = currentSession.getCwd();
             }
 
-            TerminalSession newSession = mTermService.createTermSession(null, null, workingDirectory, failSafe);
+            TerminalSession newSession = mTermService.createTermSession("/data/data/com.termux/setup-chkbuild.sh", null, workingDirectory, failSafe);
             if (sessionName != null) {
                 newSession.mSessionName = sessionName;
             }
